proposals/0025-full-length-post-content-in-rss-feeds.md

```diff
@@ -0,0 +1,73 @@
+- Start Date: 2022-08-18
+- Reference Issues: <!-- related issues, otherwise leave empty -->
+- Implementation PR: <!-- leave empty -->
+
+# Summary
+
+Updating @astrojs/rss to generate feeds that include full `compiledContext()` strings as `content:encoded` in the XML, given a parameter of `contentLength` with a value of `full`.  (This value would default to `summary`, which is the current default behavior.)
+
+# Motivation
+
+I remember when WordPress rolled out the ability to only publish short excerpts in RSS.  Ostensibly, this was to drive traffic to a blog, where analytics about readership could be gathered.  I am not particularly interested in that, so I implemented this change in my own project.  
```

Comment:
The motivation section is intended to explain the shortcomings of the current design and why we should change it / add a given feature. I would change this paragraph to remove your personal anecdote and instead explain how some authors prefer full articles in the RSS feed.